bundle/config/loader/process_root_includes.go

```diff
@@ -71,11 +71,11 @@ func (m *processRootIncludes) Apply(ctx context.Context, b *bundle.Bundle) diag.
 				continue
 			}
 			seen[rel] = true
-			if filepath.Ext(rel) != ".yaml" && filepath.Ext(rel) != ".yml" {
+			if filepath.Ext(rel) != ".yaml" && filepath.Ext(rel) != ".yml" && filepath.Ext(rel) != ".json" {
 				diags = diags.Append(diag.Diagnostic{
 					Severity:  diag.Error,
-					Summary:   "Files in the 'include' configuration section must be YAML files.",
```

Comment:
You also need to change an output for the tests here to make sure it matches: https://github.com/databricks/cli/blob/main/acceptance/bundle/includes/non_yaml_in_include/output.txt

Reply:
thank you, changed the output